openlibrary/macros/LocateButton.html

```diff
@@ -0,0 +1,9 @@
+$def with(edition_key)
+
+$ locateUrl = "/books/XXX/-/borrow?action=locate".replace('XXX', edition_key or '')
+
+<a class="cta-btn cta-btn--available cta-btn--w-icon cta-btn--external" href="$locateUrl"
+  data-ol-link-track="CTACLICK|Locate">
+  <span class="btn-icon map"></span>
```

Comment:
What's this span for?

Reply:
Ah that's for when the button is in the read dropdown, it will have a little map icon. I can move it to the read dropper branch.